x-pack/plugin/inference/qa/test-service-plugin/src/main/java/org/elasticsearch/xpack/inference/mock/TestDefaultInferenceServiceExtension.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.inference.mock;
+
+import org.elasticsearch.action.ActionListener;
+import org.elasticsearch.inference.InferenceService;
+import org.elasticsearch.inference.InferenceServiceExtension;
+import org.elasticsearch.inference.MinimalServiceSettings;
+import org.elasticsearch.inference.Model;
+import org.elasticsearch.inference.TaskType;
+
+import java.util.List;
+
+public class TestDefaultInferenceServiceExtension implements InferenceServiceExtension {
+
+    @Override
+    public List<Factory> getInferenceServiceFactories() {
+        return List.of(TestDefaultInferenceService::new);
+    }
+
+    public static class TestDefaultInferenceService extends TestSparseInferenceServiceExtension.TestInferenceService {
+
+        private static final String DEFAULT_EIS_ELSER_INFERENCE_ID = ".elser-2-elastic";
+        private static final String DEFAULT_FALLBACK_ELSER_INFERENCE_ID = ".elastic-inference";
```

Comment:
Curious why we're re-defining default values here instead of re-using the real values?